bytecomp/bytelibrarian.ml

```diff
@@ -99,8 +111,17 @@ let create_archive file_list lib_name =
        output_binary_int outchan 0;
        let units =
          List.flatten(List.map (copy_object_file outchan) file_list) in
```

Comment:
I think if we used `rev_map` here, we wouldn't have to `rev` the list below?

Reply:
`units` is used in its normal order bellow. We could rev_map here and rev_map again below but it would be confusing.

Reply:
Got it, yeah, multiple `rev`s would be confusing/error-prone